app/src/main/java/cookpad/com/bottomnavwatson/DetailFragment.kt

```diff
@@ -0,0 +1,60 @@
+package cookpad.com.bottomnavwatson
+
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.content.Context
+import android.os.Bundle
+import android.view.View
+import androidx.core.app.NotificationCompat
+import androidx.core.app.NotificationManagerCompat
+import androidx.fragment.app.Fragment
+import androidx.navigation.NavDeepLinkBuilder
+import androidx.navigation.fragment.NavHostFragment.findNavController
+import androidx.navigation.ui.AppBarConfiguration
+import androidx.navigation.ui.setupWithNavController
+import kotlinx.android.synthetic.main.detail_fragment.*
+import java.util.*
+
+class DetailFragment : Fragment(R.layout.detail_fragment) {
+    companion object {
+        private const val CHANNEL_ID = "1"
+    }
```

Comment:
[nit] I would suggest having companion class at the bottom of the class as per guidelines

Reply:
Done -> https://github.com/cookpad/BottomNavWatson/pull/4/commits/f6d7e53ad2d2cd4bd5e2f0c4e28d6d3f131b05c6